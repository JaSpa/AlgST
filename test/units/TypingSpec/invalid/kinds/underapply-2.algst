forall (p:P). D3 p
