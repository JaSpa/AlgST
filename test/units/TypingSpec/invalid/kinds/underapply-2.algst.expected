
1:15: error:
	 Invalid type application
	    D3 p
	 Type constructor D3 needs 3 paramters, 1 provided.